### PR TITLE
Assist pipeline language voice

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Needs https://github.com/home-assistant/core/pull/91619

CleanShot 2023-04-20 at 16 29 17

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-tts.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  ): string =>
+    this.hass.localize(
+      `ui.panel.config.voice_assistants.assistants.pipeline.detail.form.${schema.name}`
+    );
+
+  protected render() {
+    return html`
+      <div class="section">
+        <div class="intro">
+          <h3>Text to speech</h3>
```
Comment (balloob) [inline src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-tts.ts:60]: We are going to follow the Microsoft style guideline and write it with dashes.
```suggestion
          <h3>Text-to-speech</h3>
```

File: src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-tts.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+          <p>An engine does something that needs some explanation.</p>
```
Comment (balloob) [inline src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-tts.ts:61]: When you are using the pipeline as a voice assistant, the text-to-speech engine turns the conversation text responses into audio.

File: src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-stt.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  ): string =>
+    this.hass.localize(
+      `ui.panel.config.voice_assistants.assistants.pipeline.detail.form.${schema.name}`
+    );
+
+  protected render() {
+    return html`
+      <div class="section">
+        <div class="intro">
+          <h3>Speech to text</h3>
```
Comment (balloob) [inline src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-stt.ts:53]:
```suggestion
          <h3>Speech-to-text</h3>
```

File: src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-stt.ts [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+          <p>An engine does something that needs some explanation.</p>
```
Comment (balloob) [inline src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-stt.ts:54]: When you are using the pipeline as a voice assistant, the speech-to-text engine turns your voice command into text.

**Comment by piitaya** [comment]
Rebased and wording updated. PR will follow for stt language, tts language and tts voice selector.

**Review by balloob — APPROVED** [review]

**Pull request merged**